### DISCUSSION TITLE
Ionospheric phase in topsStack

### DISCUSSION THREAD

**Discussion by bjmarfito** [isce-framework/isce2#706] [Q&A] [answered]
Hi @CunrenLiang,

I would like to ask if there is a way to automatically apply the estimated ionospheric phase in topsStack. I am asking this since the interferogram workflow with ionospheric estimation does not apply it automatically.

Best,
Bryan

**Comment by CunrenLiang** [comment, marked as answer]
No, since this is very simple. The result is ionospheric phase of each date. Ionospheric correction can be easily done by imageMath.py in ISCE.

**Comment by bjmarfito** [comment]
Is it this code Cunren, `imageMath.py -e='a*exp(-1.0*J*b)' --a=<original interferogram> --b=<estimated ionospheric phase> -s BIP -t cfloat -o <corrected interferogram>` ? Please correct me if I'm wrong.

**Reply by CunrenLiang** [reply]
Yes, but you need to change it accordingly if the ionospheric phase is for a single date. You can also subtract ionospheric phase from unwrapped interferogram.

**Reply by bjmarfito** [reply]
Got it. Thank you very much Cunren!
